### DISCUSSION TITLE
请问有钉钉开发群吗

### DISCUSSION THREAD

**Discussion by wefiy** [alibaba/formily#1209] [Q&A 2.x] [answered]
请问有钉钉开发群吗

**Comment by janryWang** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/4060976/114341402-43ca7300-9b8c-11eb-8d60-cfa908f2130e.png)

**Comment by 1owrie** [comment]
加不进去了怎么办呢 满1000人了

**Reply by zexliu** [reply]
成员已达上限....
